### ISSUE TITLE
Better feedback during and after firmware update

### ISSUE THREAD

**Issue by cpfeiffer** [Freeyourgadget/Gadgetbridge#271]
We currently display toasts during the firmware update, but afterwards, the user is left without any hints whether firmware update finished successfully or not. 

We should display the results in the update screen.

**Comment by danielegobbetti** [comment]
the notification does display the status as completed and the progressbar should reach 100% though :)

**Comment by cpfeiffer** [comment]
That's true,  but apparently users don't find it and/or we should display intermediate messages that are otherwise lost as toasts.
